rhel_osv/csaf.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) 2024 Jason Shepherd
+# License: GPLv3+
+
+import json
+import logging
+from pathlib import Path
+from typing import Any
+
+logger = logging.Logger(__name__)
+
+
+class Reference(object):
+    """
+    class to handle CSAF References
+    """
+    def __init__(self, csaf_ref: dict[str, str]):
+        self.category = csaf_ref["category"]
+        self.summary = csaf_ref["summary"]
+        self.url = csaf_ref["url"]
+
+
+class Remediation(object):
+    """
+    class to handle remediation advise in CSAF data
```

Comment:
Fix spelling mistake

```suggestion
    class to handle remediation advice in CSAF data
```

Reply:
updated.